src/library/librarywatcher.cpp

```diff
@@ -764,14 +780,15 @@ QString LibraryWatcher::PickBestImage(const QStringList& images) {
 }
 
 QString LibraryWatcher::ImageForSong(const QString& path,
-                                     QMap<QString, QStringList>& album_art) {
+                                     QMap<QString, QStringList>& album_art,
```

Comment:
This should really be a pointer while you're here.

Reply:
Fixed.